source/draw/gpu/opengl/OpenGLBgra.ooc

```diff
@@ -18,11 +18,11 @@
 use ooc-math
 use ooc-draw
 use ooc-draw-gpu
-import OpenGLES3/Texture, OpenGLES3Canvas, OpenGLES3Texture
+import backend/gles3/Texture, OpenGLCanvas, OpenGLTexture
```

Comment:
It will not be removed but instead import backend/GLTexture or something similar.